PhysicsTools/PatAlgos/python/slimming/isolatedTracks_cfi.py

```diff
@@ -58,10 +66,16 @@
 
     pfneutralsum_DR = cms.double(0.05),
 
+    useHighPurity = cms.bool(False),
```

Comment:
this change is already present in the release
https://github.com/cms-sw/cmssw/blob/b305e3a2a97c8c4e6de61a827dc76cda31713f47/PhysicsTools/PatAlgos/python/slimming/isolatedTracks_cfi.py#L61
this is apparently because the base commit is from back in July of 2020 b389eeac191cee9f1ce0d231df3f00b36bf154af

please rebase
see e.g. http://cms-sw.github.io/tutorial-resolve-conflicts.html
`git cms-rebase-topic`